pkg/gomod/parse_test.go

```diff
@@ -0,0 +1,75 @@
+package gomod
+
+import (
+	"os"
+	"path"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/kylelemons/godebug/pretty"
+)
+
+func TestParse(t *testing.T) {
+	vectors := []struct {
+		file      string
+		libraries []types.Library
+	}{
+		{
+			file:      "testdata/gomod_normal.sum",
+			libraries: GoModNormal,
+		},
+		{
+			file:      "testdata/gomod_many.sum",
+			libraries: GoModMany,
+		},
+		{
+			file:      "testdata/gomod_trivy.sum",
+			libraries: GoModTrivy,
+		},
+	}
+
+	for _, v := range vectors {
+		t.Run(path.Base(v.file), func(t *testing.T) {
+			f, err := os.Open(v.file)
+			if err != nil {
+				t.Fatalf("Open() error: %v", err)
+			}
```

Comment:
```suggestion
			require.NoError(t, err)
```

Recently, we migrated to `github.com/stretchr/testify/assert` and `github.com/stretchr/testify/require`. Could you rewrite this test using them like the following?
https://github.com/aquasecurity/go-dep-parser/blob/main/pkg/jar/parse_test.go

Reply:
updated, rewrote the rest of the tests as well

Reply:
Great job! Thanks!